### DISCUSSION TITLE
mc-server  Error: Unable to access jarfile paper.jar

### DISCUSSION THREAD

**Discussion by Jillert13** [AlexProgrammerDE/balena-minecraft-server#172] [Q&A]
I know that this issue has been addressed many times by looking through the other discussions, but I have not been able to solve it in my case as of yet. I have this issue and the server keeps looping, I think in an attempt to self correct, I'm new to balena so I'm not entirely sure. I have attempted to create a new variable ENABLE_UPDATE set to true with no change. I appreciate any help that you can provide.
![jarfile error](https://user-images.githubusercontent.com/93365450/139378361-c759c85b-f44f-4579-b4da-686a577b0f10.png)

**Comment by viedok** [comment]
I think you can now mark it as resolved 😉.
